Core/src/Geometry/GeometryIdentifier.cpp

```diff
@@ -24,7 +24,7 @@ std::ostream& Acts::operator<<(std::ostream& os, Acts::GeometryIdentifier id) {
                                               id.sensitive(), id.extra()};
 
   bool writeSeparator = false;
-  for (auto i = 0u; i < 5u; ++i) {
+  for (auto i = 0u; i < 6u; ++i) {
```

Comment:
could we do something like `sizeof(levels)`?

might be worth to use `std::array` up there and enforce same size